packages/smooth_app/lib/pages/preferences_v2/app_bars/app_bar_background.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+
+class AppBarBackground extends StatelessWidget {
```

Comment:
Your custom `AppBar` can be used elsewhere.
I would move it to the global folder instead.

Also I recommend changing the name

Reply:
I've moved my `AppBar` to the generic_lib. I'm not sure how to rename them, they are quite specific to the authentication state.